emulated_hue/api.py

```diff
@@ -670,7 +670,7 @@ async def __async_light_action(self, entity: dict, request_data: dict) -> None:
             )
 
         # execute service
-        await self.hass.async_call_service(const.HASS_DOMAIN_LIGHT, service, data)
+        await self.hass.call_service(const.HASS_DOMAIN_LIGHT, service, data)
```

Comment:
Why was this changed to remove async while it's still being awaited?

Reply:
You asked me a while ago to remove the async_ prefixes so I started here ;-)
These async_ prefixes come from a time asyncio was not mainstream. I'm slowly adjusting my code to not include the prefix anymore...

Reply:
Ah right! I thought you had good reason for leaving them in at the time. Typically my IDE will tell me when I'm calling an async method without await, so it was never a huge deal with or without the. I'm fine either way but thought it was a little strange given that you previously mentioned it being important while removing it here.

Reply:
I do not have a preference anymore. I liked the async_ prefix to make it more clear it's async and to allow a method to have both a sync and a async version. I guess that now we're actually only coding everything async the prefix does no longer make sense. Asyncio even warns you nowadays if you're calling an async method without awaiting it.

Maybe turn things around and if there's a sync method available for some logic, just prefix it with "sync_"

Reply:
I don't think adding sync is absolutely necessary ;), but we may look into adding unit tests that could catch issues in the code like this if they arise. I'm going to handle all the dependabot PRs and merge this after. 

Perhaps it's time for a small release after as well.